admin-ui/src/test/java/io.crate/rest.action.admin/AdminUIHttpIntegrationTest.java

```diff
@@ -0,0 +1,73 @@
+package io.crate.rest.action.admin;
+
+
+import org.apache.http.Header;
+import org.apache.http.client.methods.*;
+import org.apache.http.client.protocol.HttpClientContext;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+import org.elasticsearch.common.transport.InetSocketTransportAddress;
+import org.elasticsearch.http.HttpServerTransport;
+import org.junit.Before;
+
+import java.io.IOException;
+import java.net.InetSocketAddress;
+import java.net.URI;
+import java.util.*;
+import java.util.concurrent.ExecutionException;
+
+import static org.hamcrest.core.Is.is;
+
+
+public abstract class AdminUIHttpIntegrationTest extends AdminUIIntegrationTestBase {
```

Comment:
Would it make sense to integrate the stuff from AdminUIIntegrationTestBase into this class as well?
There is only 1 usage so far.